devel/libbsd/Portfile

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+
+name            libbsd
+categories      devel
+
+description     library for utility functions from BSD systems
+long_description \
+                This library provides useful functions commonly found on BSD systems, \
+                and lacking on others like GNU systems.  Thus making it easier to port \
+                projects with strong BSD origins, without needing to embed the same code \
+                over and over again on each project.
+
+homepage        https://libbsd.freedesktop.org/wiki/
+
+platforms       darwin
+license         MIT BSD ISC
+maintainers     @Zweihorn openmaintainer
+
+
+subport libbsd-devel {}
+
+if {${subport} eq ${name}} {
+
+    # This is the libbsd default
+
+    known_fail      yes
+
+    version         0.11.7
+
+    revision        0
+
+    master_sites    https://libbsd.freedesktop.org/releases/
+
+    use_xz          yes
+
+    checksums       rmd160  76c606f4e2c6bfce1fe9fff221de23a14b6887a7 \
+                    sha256  9baa186059ebbf25c06308e9f991fda31f7183c0f24931826d83aa6abd8a0261 \
+                    size    418508
+
+    conflicts       libbsd-devel
+
+} else {
+
+    # This is the libbsd-devel subport for Development
+
+    long_description {*}${long_description} \
+        This port provides the devel version of ${name}, as appropriate.
+
+    version         0.11.7-dirty
```

Comment:
I've never seen any other port use the suffix `-dirty` on a version number.

Reply:
After some consideration with `gitlab.version` I changed to the explicitly set `version 2023-08-10` (and later dates).

```
  libbsd-devel @0.11.7-dirty_0
  libbsd-devel @73b25a8f871b3a20f6ff76679358540f95d7dbfd_0
  libbsd-devel @2023-08-10_0 (active)
```

Reply:
Granted.